### DISCUSSION TITLE
Group dates for long period visualizations

### DISCUSSION THREAD

**Discussion by L0bit0** [marcusolsson/grafana-hourly-heatmap-panel#77] [Q&A]
Hi

I have created a heat map for my home temperatures. In some cases I should long periods (90 days, 1 year...)

when I do, this dates overlap and is impossible to see:
![image](https://user-images.githubusercontent.com/10359623/232754278-f1518802-f626-4938-9474-f4468702a203.png)

There is a way to group by month, day... in the date label only?

Thanks

**Comment by maxdraki** [comment]
I have precisely the same issue - for pretty much the same use case (home energy consumption in my case) and I would like a way to change the date format on the time axis if possible to prevent them overlapping. Thanks
